conda_smithy/ci_register.py

```diff
@@ -220,9 +226,6 @@ def add_project_to_travis(user, project):
             print(' * Registered on travis-ci')
         else:
             raise RuntimeError('Unable to register on travis-ci, response from hooks was negative')
-        url = '{}/users/sync'.format(endpoint)
-        response = requests.post(url, headers=headers)
-        response.raise_for_status()
```

Comment:
Also contacted him offline about this. FWIW I agree with you that this is probably not needed.